libs/common/src/assets/i18n/en.json

```
@@ -197,21 +198,24 @@
 	"billing.phone": "Phone number",
 	"billing.phone_placeholder": "e.g. +34 000 000 000",
 	"billing.postal_code": "ZIP",
-	"billing.quotas.answers": "Generative answer",
 	"billing.quotas.files": "Maximum number of files",
 	"billing.quotas.files-up-to": "Files up to",
+	"billing.quotas.generative": "Generative answer",
 	"billing.quotas.ingestion": "Ingestion",
 	"billing.quotas.kbs": "Number of knowledge boxes",
-	"billing.quotas.medias": "Video & audio indexing ",
-	"billing.quotas.paragraphs": "Number of paragraph",
-	"billing.quotas.paragraphs_stored": "Paragraph stored",
+	"billing.quotas.media": "Video & audio indexing ",
```

I guess the extra space isn't in purpose?
```suggestion
	"billing.quotas.media": "Video & audio indexing",
```

---

Fixed